components/server/ee/src/workspace/workspace-starter.ts

```diff
@@ -24,7 +24,7 @@ export class WorkspaceStarterEE extends WorkspaceStarter {
         user: User,
         excludeFeatureFlags: NamedWorkspaceFeatureFlag[],
         ideConfig: IDEConfig,
-        forcePVC: boolean,
+        PVCEnabledForPrebuilds: boolean,
```

Comment:
nit: function args are always lower case. So this would be `pvcEnabledForPrebuilds`. :+1: